src/config/configuration.ts

```diff
@@ -1,18 +1,28 @@
+import {
+  getEnv as baseGetEnv,
+  getNodeEnv,
+  toInteger,
+} from '@fullstacksjs/toolbox';
 import { registerAs } from '@nestjs/config';
 import Joi from 'joi';
 
-interface IAppConfig {
-  nodeEnv: string;
-  port: number;
+const getEnv = baseGetEnv<Env>;
```

Comment:
Here we use the `Env` that we declare in `environment.d.ts` to make this function typesafe.